shared/rakefiles/ci.rake

```diff
@@ -9,8 +9,6 @@ task :set_vars_ci => [:set_vars] do
     " run ",
     " run --volume #{ENV["HOME"]}/.aws:/aws-backup ",
   )
-  # This duplicates information in docker-compose.yaml,
```

Comment:
Please retain this comment, and update it to say the duplication is now in shared/rakefiles/xk_config.rake.

Or, you could fix the duplicate definition.

Reply:
Comment returned.